java-core/google-cloud-core/src/main/java/com/google/cloud/ServiceOptions.java

```diff
@@ -767,4 +799,52 @@ public String getClientLibToken() {
   public String getQuotaProjectId() {
     return quotaProjectId;
   }
+
+  /**
+   * Returns the resolved host for the Service to connect to Google Cloud
+   *
+   * <p>The resolved host will be in `https://www.{serviceName}.{resolvedUniverseDomain}/` format.
```

Comment:
Would you add a reference about adding "www."? I checked `www.speech.googleapis.com` but it (the DNS entry) does not exist.

```
suztomo@suztomo2:~$ ping www.speech.googleapis.com
ping: www.speech.googleapis.com: Name or service not known
```

Reply:
Added a link to DEFAULT_HOST

Reply:
This works for `cloudkms.googleapis.com` not sure why there is an issue for `speech.googleapis.com`